src/app/core/components/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1 +1,4 @@
-<p>breadcrumbs works!</p>
+<div *ngFor="let breadcrumb of breadcrumbs; let i = index">
+    <span class="bc-item" [class.active]="i+1==breadcrumbs.length">{{breadcrumb}}</span>
```

Comment:
There is a practice  to move all JS from templates to components, create variable` this.isBreadCrumbActive`  and do all the job there, in template just use isBreadCrumbActive

Reply:
===?